traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -701,6 +701,18 @@ func UsernameExists(uname string, tx *sql.Tx) (bool, error) {
 	return exists, err
 }
 
+// GetTypeIDByName reports whether or not a type exists with the given name and the id of that type.
```

Comment:
nit: the ordering of this `func`'s returns is different in the GoDoc than in the actual code.